universal-login-react/src/ui/commons/ErrorMessage.tsx

```diff
@@ -0,0 +1,23 @@
+import React, {ReactNode} from 'react';
+import './../styles/errorMessage.sass';
+import './../styles/errorMessageDefault.sass';
+import {getStyleForTopLevelComponent} from '../../core/utils/getStyleForTopLevelComponent';
+
+interface ErrorMessageProps {
+  title?: string;
+  message?: string;
+  children?: ReactNode;
+  className?: string;
+}
+
+export const ErrorMessage = ({title = 'Something went wrong', message = 'Please try again later.', children, className}: ErrorMessageProps) => (
+  <div className={getStyleForTopLevelComponent(className)}>
```

Comment:
Wrap it in ``<div className="universal-login-error-message">...</div>``

Reply:
Done.